src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -158,13 +158,11 @@ protected HttpMessage newFullBodyMessage(ByteBuf body) {
 		HttpResponse res =
 				new DefaultFullHttpResponse(version(), status(), body);
 
-		if (!HttpMethod.HEAD.equals(method())) {
```

Comment:
@violetagg , do you have a context of why this condition was written in the first place?

Reply:
it is just a bug